src/pages/Metrics/locales/zh-CN.ts

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
And you should import those i18n files in `/src/locales/xxx`, please have a look at SSL page.

Reply:
Thanks, I have added, but before I added this, it is ok too, why?

Reply:
Not sure, please check the locales under /src to see if there have already imported those module locales?